app/views/admin/users/edit.html.erb

```diff
@@ -48,7 +48,7 @@
   </div>
 
 <%= render "notes" %>
-<%= render "negative_reactions" %>
```

Comment:
I can't see this partial referenced anywhere else. If it's no longer in use maybe worth getting rid of it?

Reply:
Great question, @fdoxyz ! This partial is used to display all of the `vomit` reactions that a particular user has on the `/admin/users/:id/edit` page just like the `reports` partial displays all of the recent `FeedbackMessage`s for a particular user on that page. This code is (unfortunately) showing up in this PR's history due to a rebase--I originally started building out the `_negative_reactions` partial in this PR and then pulled it out into a separate one which has since been merged/deployed. 😅

Reply:
Got it, thanks for clarifying!

Reply:
Absolutely! Thanks for the review. 😄